Android crash after changing language

Hi there! Bit of a weird/edge-case crash that I've discovered that I was wondering if you would know anything about.

Steps to reproduce:
1. Open the app (I used the zoom sample as an example)
2. Background the app
3. Change your language in settings
4. Open the app again

After doing that, I see the following crash:

```
FATAL EXCEPTION: main
Process: com.zoom, PID: 3187
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.zoom/com.zoom.MainActivity}: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment com.navigation.reactnative.SceneFragment: could not find Fragment constructor
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3488)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3635)
	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:5582)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5490)
	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:69)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2175)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:7860)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1075)
Caused by: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment com.navigation.reactnative.SceneFragment: could not find Fragment constructor
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:462)
	at androidx.fragment.app.FragmentContainer.instantiate(FragmentContainer.java:50)
	at androidx.fragment.app.FragmentState.instantiate(FragmentState.java:80)
	at androidx.fragment.app.FragmentManagerImpl.restoreAllState(FragmentManager.java:3109)
	at androidx.fragment.app.FragmentController.restoreAllState(FragmentController.java:158)
	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:344)
	at androidx.appcompat.app.AppCompatActivity.onCreate(AppCompatActivity.java:85)
	at com.facebook.react.ReactActivity.onCreate(ReactActivity.java:43)
	at android.app.Activity.performCreate(Activity.java:7955)
	at android.app.Activity.performCreate(Activity.java:7944)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1307)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3463)
	... 13 more
Caused by: java.lang.NoSuchMethodException: com.navigation.reactnative.SceneFragment.<init> []
	at java.lang.Class.getConstructor0(Class.java:2332)
	at java.lang.Class.getConstructor(Class.java:1728)
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:443)
	... 24 more
```

--- a/NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/NavigationStackView.java
+++ b/NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/NavigationStackView.java
@@ -124,14 +124,19 @@ public class NavigationStackView extends ViewGroup implements LifecycleEventList
     public static class StackFragment extends Fragment {
         private NavigationStackView stack;
 
+        public StackFragment() {
+            super();
+        }
+
         StackFragment(NavigationStackView stack) {
+            super();
             this.stack = stack;
         }
 
         @Nullable
         @Override
         public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
-            return stack;
+            return stack != null ? stack : new View(getContext());
         }
     }
 }

--- a/NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/SceneFragment.java
+++ b/NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/SceneFragment.java
@@ -16,6 +16,10 @@ import java.util.HashSet;
 public class SceneFragment extends Fragment implements SharedElementContainer {
     private SceneView scene;
 
+    public SceneFragment() {
+        super();
+    }
+
     SceneFragment(SceneView scene, HashSet<String> sharedElements) {
         super();
         this.scene = scene;
@@ -27,11 +31,14 @@ public class SceneFragment extends Fragment implements SharedElementContainer {
     @Nullable
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
-        if (scene.getParent() != null)
-            ((ViewGroup) scene.getParent()).endViewTransition(scene);
-        if (scene.transitioner != null)
-            postponeEnterTransition();
-        return scene;
+        if (scene != null) {
+            if (scene.getParent() != null)
+                ((ViewGroup) scene.getParent()).endViewTransition(scene);
+            if (scene.transitioner != null)
+                postponeEnterTransition();
+            return scene;
+        }
+        return new View(getContext());
     }
 
     @Override

--- a/NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/TabBarView.java
+++ b/NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/TabBarView.java
@@ -228,6 +228,10 @@ public class TabBarView extends ViewPager {
         TabBarItemView tabBarItem;
         View view;
 
+        public TabFragment() {
+            super();
+        }
+
         TabFragment(TabBarItemView tabBarItem) {
             super();
             this.tabBarItem = tabBarItem;
@@ -239,7 +243,7 @@ public class TabBarView extends ViewPager {
         @Nullable
         @Override
         public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
-            return view;
+            return view != null ? view : new View(getContext());
         }
     }
 

